src/lib/serverUtils.test.ts

```
@@ -0,0 +1,354 @@
+      const mockOrdiscanInstance = {} as Ordiscan;
+      MockedOrdiscan.mockImplementation(() => mockOrdiscanInstance);
+
+      const client = getOrdiscanClient();
+
+      expect(MockedOrdiscan).toHaveBeenCalledWith('test-ordiscan-key');
+      expect(client).toBe(mockOrdiscanInstance);
+    });
+
+    it('should throw error when ORDISCAN_API_KEY is missing', () => {
+      // Ensure API key is not set
+      delete process.env.ORDISCAN_API_KEY;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)